### PR TITLE
20220929 minor fixes

### PR DESCRIPTION
**Pull request by lassoan**

### CODE REVIEW

**Review by cpinter — DISMISSED** [review]
Great, thank you!

File: Modules/Loadable/Markups/MRML/vtkMRMLMarkupsLineNode.h [hunk 1]
```diff
@@ -75,6 +75,48 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsLineNode : public vtk
   /// Return line length (distance between the two line endpoints) in world coordinate system.
   virtual double GetLineLengthWorld();
 
+  //@{
+  /// Legacy method from vtkMRMLAnnotationRulerNode
+  /// Get line start/end point in local coordinate system.
+  /// Returns true if the point position is defined.
+  bool GetPosition1(double position[3]);
+  bool GetPosition2(double position[3]);
+  //@}
```
[the remark this answers is not in the document]
Comment (lassoan) [reply Modules/Loadable/Markups/MRML/vtkMRMLMarkupsLineNode.h]: OK, it makes sense. I've now added methods that are as convenient to use as the legacy methods (e.g., you can add start and end point of the line in arbitrary order), but have somewhat better and more consistent names; and made the methods that are just added for background compatibility as deprecated.

[the remark this answers is not in the document]
Comment (jamesobutler) [reply Modules/Loadable/Markups/MRML/vtkMRMLMarkupsLineNode.h]: Perfect! Thanks

**Review by jamesobutler — APPROVED** [review]

**Review by jcfr — APPROVED** [review]

**Comment by lassoan** [comment]
Thanks for the reviews!

**Pull request merged**